Assignment3/Q1_test.py

```diff
@@ -0,0 +1,49 @@
+"""Tests for question 1 - word search."""
+
+import unittest
+
+import Q1
+
+
+class Q1Test(unittest.TestCase):
```

Comment:
Sorry. I did not actually test it :)
If you want to fix that issue, you can use the `list()` function:
```
>>> x = ('a', 'b')
>>> x
('a', 'b')
>>> list(x)
['a', 'b']
>>>
```